src/watcher.js

```diff
@@ -31,8 +31,8 @@ Watcher.prototype = {
         }
 
         // Create an individual watcher for this file.
-        streams[bundleFile] = gulpWatch(bundleFile, function (bundleFile) {
-          bundleFile = bundleFile.path;
+        streams[bundleFile] = gulpWatch(bundleFile, function (bundleFileVinyl) {
+          bundleFile = bundleFileVinyl.path;
```

Comment:
This change could also just be:
```js
    streams[bundleFile] = gulpWatch(bundleFile, function () {
```
and the callback can just continue to use `bundleFile` from the outer scope.